src/go/rpk/pkg/adminapi/admin.go

```diff
@@ -152,7 +152,7 @@ func licenseFeatureChecks(ctx context.Context, fs afero.Fs, cl *rpadmin.AdminAPI
 					features = append(features, f.Name)
 				}
 			}
-			msg = fmt.Sprintf("A Redpanda Enterprise Edition license is required to use enterprise features: %v. For more information, see https://docs.redpanda.com/current/get-started/licenses.", features)
+			msg = fmt.Sprintf("\nWARNING: The following Enterprise features are being used in your Redpanda cluster: %v. These features require a license. To get a license, Contact us at https://www.redpanda.com/contact. For more information, see https://docs.redpanda.com/current/get-started/licenses/#redpanda-enterprise-edition\n", features)
```

Comment:
Have you checked if this '\n' is not resolved into the url?

Reply:
Yes, confirmed :+1: 
![image](https://github.com/user-attachments/assets/6b775f20-dd98-49e9-91fd-f46ab5ed754c)